### DISCUSSION TITLE
CLIO V 2020 EASYLINK

### DISCUSSION THREAD

**Discussion by zecabg** [cedricp/ddt4all#1149] [Q&A]
Hi all 
I wish to make test bench for Bosch EASYLINK unit  
Is it possible ot help ?
Which modul need to start unit on bench 
1. Radio navi main unit  - pin out 
2. Display  - pin out 
3. Can data from  - BCM  - pin out 
4. Instrument cluster - pin out 
 Any info will help ...
 Thanks  in advance 
Zeca

**Comment by Clob63** [comment]
> Bonjour à tous, je souhaite fabriquer un banc de test pour une unité Bosch EASYLINK. Est-ce possible ? Pouvez-vous m'aider ? Quel module faut-il pour démarrer l'unité sur le banc ?
> 
> 1. Unité principale de navigation radio - brochage
> 2. Affichage - brochage
> 3. Les données peuvent provenir du brochage du BCM.
> 4. Tableau de bord - brochage. 
>    Toute information sera utile... 
>    Merci d'avance, 
>    Zeca.



Hi, were you able to find this information for this project?
